### PR TITLE
Add nicer frontend, fix aurora issue

### PR DESCRIPTION
**Pull request by mikedotexe**
This is one of the tasks I had some agency folks work on to get a hang of our system. There was an issue with the Aurora pool, and the frontend looks quite good.

This is actually a very useful site, so having it look fresh is a plus.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -132,7 +131,7 @@ async function lookupLockup(near, accountId) {
   } catch (error) {
     console.warn(error);
     return {
-      lockupAccountId: `${lockupAccountId} doesn't exist`,
```
Comment (telezhnaya) [inline script.js]: How will it look with this change if the account does not have the corresponding lockup?

**Review by frol — CHANGES_REQUESTED** [review]
Layout gets broken when I use any account with a lockup:

image

**Comment by mikedotexe** [comment]
> Layout gets broken when I use any account with a lockup:

Thanks, Frol. Definitely an oversight of mine. Here's what the new code looks like for `xkv.near`:

Screen Shot 2022-02-10 at 3 58 41 PM

File: script.js (hunk 1 above)
Comment (mikedotexe) [reply script.js]: Good eye, Olga, thank you.
I've fixed that here:
https://github.com/near/account-lookup/pull/17/commits/58d51c45f506782c1483322094ed3b5049ef65e8

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
@mikedotexe How is it different from #17?

**Comment by mikedotexe** [comment]
> @mikedotexe How is it different from #17?

good point, closing, and the final PR to fix the Aurora issue is here:
https://github.com/near/account-lookup/pull/18

**Pull request closed without merging**